server/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/line/reader/impl/CobolLineReaderImpl.java

```diff
@@ -46,14 +44,31 @@
 @Slf4j
 public class CobolLineReaderImpl implements CobolLineReader {
   private static final int INDICATOR_AREA_INDEX = 6;
-  private static final Pattern FIXED_FORMAT_PATTERN =
-      compile("^(.{6})" + INDICATOR_FIELD + "(.{0,4})(.{0,61})(.{0,8})");
+  private static final int MAX_LINE_LENGTH = 80;
+  private static final String LONG_LINE_MSG = "Source text can not go past column 80.";
+  private static final String INCORRECT_LINE_FORMAT_MSG = "Unexpected indicator area content";
+  private static final Map<String, CobolLineTypeEnum> INDICATORS =
+      Map.of(
+          "*",
+          COMMENT,
+          "/",
+          COMMENT,
```

Comment:
Lines like:
```
"*", COMMENT,
"/", COMMENT,
```
will be easy to read.

Reply:
It's true, but this is just an automated alignment